bin/schema_generator.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# TODO: Update schema mapping for validation
+# TODO: Handle overwriting glean schemas
+
+GCS_BUCKET="mozilla-generated-schemas"
+MPS_BRANCH="generated-schemas"
+
+# 0. Install dependencies
+
+cargo install --git https://github.com/acmiyaguchi/jsonschema-transpiler --branch 0.2 --force
+
+virtualenv mgs-venv 
+source mgs-venv/bin/activate
+pip install -U mozilla-schema-generator
+
+# 1. Pull in all schemas from MPS
+
+SCHEMAS_DIR="schemas"
+BASE_DIR="/Users/frankbertsch/repos/sandbox"
+
+cp metadata_merge.py $BASE_DIR
+cd $BASE_DIR
+rm -rf mozilla-pipeline-schemas
+
+git clone https://www.github.com/mozilla-services/mozilla-pipeline-schemas
+cd mozilla-pipeline-schemas/$SCHEMAS_DIR
+git checkout ping-metadata
+
+# 2. Remove all non-json schemas (e.g. parquet)
+
+find . -not -name "*.schema.json" -type f | xargs rm
```

Comment:
Is this necessary? I can image a future in which we have a unified branch (master) that is updated by both humans and machines, where parquet schemas are simply ignored in the GCP pipeline.

Reply:
Not really necessary, but I would prefer to only have the relevant schemas available. Why don't we add the parquet schemas back in later if they become useful?

Reply:
Basically my goal is to have an easy way of saying "what is running on prod"? And the parquet schemas are not included in that (for GCP).

Reply:
This is fine for now, since the generated branch will be gcp-only. I'll want to revisit this in the hypothetical future where we have a single branch. Since the responsibility of this script will not include uploading to GCS the deletion of parquetmr files could still happen farther down the pipeline.